aquarius/events/processors.py

```diff
@@ -8,12 +8,10 @@
 import os
 from abc import ABC
 from datetime import datetime
-from hashlib import sha256
 
 import requests
 from jsonsempai import magic  # noqa: F401
 
-from aquarius.app.auth_util import compare_eth_addresses
```

Comment:
so should we change all instances of compare_eth_addressses with the new is_address_match function?

Reply:
no, it's just not used in that file